ha/note.c

```diff
@@ -140,6 +140,36 @@ M0_INTERNAL void m0_ha_state_single_post(struct m0_ha_nvec *nvec)
 	m0_ha_msg_nvec_send(nvec, 0, false, M0_HA_NVEC_SET, NULL);
 }
 
+static void m0_ha_add_dynamic_fid_to_confc(
+			struct m0_conf_cache    *cache,
+			struct m0_conf_obj      *base_obj,
+			const struct m0_ha_note *nv_note,
+			uint64_t                 ignore_same_state)
+{
+	struct m0_conf_obj     *new_obj;
+        enum m0_ha_obj_state    prev_ha_state;
+	int                     rc;
+
+	M0_LOG(M0_DEBUG, "Received fid: "FID_F" look up fid :"FID_F,
+		FID_P(&nv_note->no_id), FID_P(&base_obj->co_id));
+
+	if (m0_fid_tget(&nv_note->no_id) == 'r') {
+		rc = m0_confc_cache_add_process(cache, base_obj,
+						&new_obj, nv_note);
```

Comment:
Just pass new fid as a last argument, and updated the state after returning from the API.

Reply:
done